src/routes/expenses.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const express = require('express');
+const expenseController = require('../controllers/expenses.js');
+
+const router = express.Router();
+
+router.get('/', expenseController.getFiltered);
+router.get('/:expenseId', expenseController.getById);
+router.post('/', express.json(), expenseController.create);
+router.patch('/:expenseId', express.json(), expenseController.update);
```

Comment:
you already handle it in the `createServer`
```suggestion
router.post('/', expenseController.create);
router.patch('/:expenseId', expenseController.update);
```